Actions/Sign/Sign.psm1

```diff
@@ -1,44 +1,98 @@
-function GetNavSipFromArtifacts
-(
-    [string] $NavSipDestination
-)
-{
-    $artifactTempFolder = Join-Path $([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+<#
+    .SYNOPSIS
+    Installs the dotnet signing tool.
+    .DESCRIPTION
+    Installs the dotnet signing tool.
+    .PARAMETER Version
+    The version of the signing tool to install.
+#>
+function Install-SignTool() {
+        param(
+            [Parameter(Mandatory = $false)]
+            [string] $Version = "0.9.1-beta.24123.2"
```

Comment:
Is a repo uses self-hosted runners and this pre-release version is installed, the signing tool will never be updated when we change this version.

This version should be in some AL-Go config where we could update it when a newer version is available (or automate it? Dependabot?) and deploy new AL-Go version if needed.

Reply:
True.. I've updated it now such that we install it locally rather than globally. That way we install the version specified **unless** the machine already has dotnet sign installed. This could be the case on some self-hosted runners if they are configured with the signing tool installed globally. That is not encouraged though.

I'll try to look into your second suggestion as well. Although setting up support for dependabot seems a bit out of scope for this PR. It's a nice idea though

Reply:
The Dependabot suggestion was more of a future enhancement. My idea was to have the version in some config, not in code, so that it's easier to update it if needed.

Reply:
I meant a config for AL-Go. So that it's clear, for example, that AL-Go v4.1 runs with v0.9.1 of the signing tool and AL-Go v5 runs with v1.0.1 of the signing tool.

Reply:
Got it - that might be a slice for later though?
I am OK with both.